src/Schema/Events/Search.ts

```diff
@@ -38,7 +38,7 @@ export interface FocusedOnSearchInput {
  *  ```
  *  {
  *    action: "selectedItemFromSearch",
- *    context_module: "priceEstimate",
+ *    context_module: "topTab",
```

Comment:
**suggestion**: `priceEstimate"` is still a valid ContextModule value and probably makes more sense paired with the rest of the example attributes here so I might remove this change.